R/Angel Chen/ticket #20492 Norman.R

```diff
@@ -0,0 +1,144 @@
+                         metadata_path = doc_path, 
+                         public = FALSE)
+#https://arcticdata.io/catalog/view/urn%3Auuid%3A8b86d811-2b0e-48ae-9f48-7a57d67eccfe
+
+pkg <- get_package(adc, 'resource_map_urn:uuid:8b86d811-2b0e-48ae-9f48-7a57d67eccfe', file_names = TRUE)
+doc <- read_eml(getObject(adc, pkg$metadata))
+emld::eml_version("eml-2.1.1")
+
+doc$dataset$coverage$geographicCoverage[[2]]$boundingCoordinates$northBoundingCoordinate <- "69.057876"
+doc$dataset$coverage$geographicCoverage[[2]]$boundingCoordinates$southBoundingCoordinate <- "69.057876"
+doc$dataset$coverage$geographicCoverage[[2]]$boundingCoordinates$westBoundingCoordinate <- "-152.862827"
+doc$dataset$coverage$geographicCoverage[[2]]$boundingCoordinates$eastBoundingCoordinate <- "-152.862827"
+
+eml_validate(doc)
+doc_path <- file.path(tempdir(), 'science_metadata.xml')
+write_eml(doc, doc_path)
+
+update <- publish_update(adc, 
+                         metadata_pid = pkg$metadata,
+                         resource_map_pid = pkg$resource_map,
+                         data_pids = pkg$data,
+                         metadata_path = doc_path, 
+                         public = FALSE)
+#https://arcticdata.io/catalog/view/urn:uuid:5b6546c3-b8a6-420f-b1ba-1422c4479f6b
+
+#https://arcticdata.io/catalog/view/urn%3Auuid%3Adda6d1ed-b817-42ad-9c23-a33b01c59a88
+
+pkg <- get_package(adc, 'resource_map_urn:uuid:160fbcbf-92af-4d3b-8ebf-6092a4435a27', file_names = TRUE)
+doc <- read_eml(getObject(adc, pkg$metadata))
+emld::eml_version("eml-2.1.1")
+
+set_rights_and_access(adc, pids = unlist(pkg), subject = 'http://orcid.org/0000-0002-5718-6032')
+
+doc$dataset$title <- "Cape Espenberg Birnirk Project (CEBP) human mitogenome summary analysis (2016-2019)"
+
+eml_validate(doc)
+doc_path <- file.path(tempdir(), 'science_metadata.xml')
+write_eml(doc, doc_path)
+
+update <- publish_update(adc, 
+                         metadata_pid = pkg$metadata,
+                         resource_map_pid = pkg$resource_map,
+                         data_pids = pkg$data,
+                         metadata_path = doc_path, 
+                         public = FALSE)
+#https://arcticdata.io/catalog/view/urn:uuid:ec4f2c29-bdb2-4927-b364-20f2d6ead811
+
+pkg <- get_package(adc, 'resource_map_urn:uuid:ec4f2c29-bdb2-4927-b364-20f2d6ead811', file_names = TRUE)
+doc <- read_eml(getObject(adc, pkg$metadata))
+emld::eml_version("eml-2.1.1")
+
+update <- publish_update(adc, 
+                         metadata_pid = pkg$metadata,
+                         resource_map_pid = pkg$resource_map,
+                         data_pids = pkg$data,
+                         use_doi=TRUE,
+                         public = TRUE)
+#https://arcticdata.io/catalog/view/doi%3A10.18739%2FA2NC5SD2M
+
+datamgmt::categorize_dataset("doi:10.18739/A2CZ32589", c("archaeology","anthropology"), "Angel")
+datamgmt::categorize_dataset("doi:10.18739/A2NC5SD2M", c("archaeology","anthropology"), "Angel")
+
```

Comment:
nice! seems like you've been able to simplify some steps down through your experience which is cool!